<launch>
    <arg name="scenario"/>
    <rosparam command="load" file="$(find supersaop)/launch/conf/$(arg scenario)"/>
    <node pkg="supersaop"
          type="situation_assessment.py"
          name="situation_assessment"
          output="screen"
          required="true"/>

    <node pkg="supersaop"
          type="observation_planning.py"
          name="observation_planning"
          output="screen"
          respawn="true"/>

    <node pkg="supersaop"
          type="ccu_bridge.py"
          name="ccu_bridge"
          output="screen"
          respawn="true"/>

    <!-- Comming soon ! -->

<!--    <node pkg="supersaop"-->
<!--          type="supervisor.py"-->
<!--          name="supervisor"-->
<!--          output="screen"-->
<!--          respawn="true"/>-->
</launch>
